configs/aaf_config.yaml

```diff
@@ -0,0 +1,27 @@
+experiment_name: aaf
+random_seed: 1998
+head_type: gaussian
+dataset:
+  type: image
+  spec: aaf
+  input_dim: 3
```

Comment:
This key is only used when the data being passed in is tabular (since the MLP we build has to have the same number of nodes in the first layer as features in the data). So you can drop `input_dim` for image data

Reply:
Fixed